public/javascripts/script.js

```diff
@@ -1,31 +1,71 @@
 // This will print in the wrong order
 // we added it for you to test to make sure data is loaded
 // 🚨🚨🚨 comment out the next 3 lines when you start working on the code
-for (let i = 0; i < mashPotatoes.length; i++) {
+//for (let i = 0; i < mashPotatoes.length; i++) {
   //addFood(steak[i], '#steak');
-  console.log(mashPotatoes[i])
-}
+ // console.log(mashPotatoes[i])
+//}
 
 // Iteration 1 using callbacks
-addFood(steak[0], '#steak', () => {
-  // ... your code here
-  addFood(steak[1], '#steak', () => {
-
-  })
-});
-
+function addSteak() {
+  addFood(steak[0], '#steak', () => {
+    addFood(steak[1], '#steak', () => {
+      addFood(steak[2], '#steak', () =>{ 
+        addFood(steak[3], '#steak', () => {
+          addFood(steak[4], '#steak', () => {
+            addFood(steak[5], '#steak', () => {
+              addFood(steak[6], '#steak', () => {
+                addFood(steak[7],'#steak', () => {
+                  table.innerHTML +=`<img src="./public/images/steak.jpg">`
+                })
+              })
+            })
+          })
+        })
+      })
+    })
+  });
+}
+//addSteak()
 
 
 // Iteration 2 using `.then()`
-addFood(mashPotatoes[0], '#mashPotatoes').then(() => {
-  // ... your code here
-  addFood(mashPotatoes[1], '#mashPotatoes')
-});
+function addMashPotatoes() {
+  addFood(mashPotatoes[0], '#mashPotatoes').then(() => {
```

Comment:
In order to make the code looks cleaner you can use this structure: https://prnt.sc/u4u573

Reply:
Same in the following lines